Project.cpp

```diff
@@ -23,8 +23,8 @@ class Note{
     string TopicName;
     string NoteContent;
     Note(string topic, string noteContent){ 
-        TopicName = topic;
-        NoteContent = noteContent; 
+        this->TopicName = topic;
```

Comment:
Consider removing `this->` when initializing class members in constructors to make the code cleaner and more readable. [medium]